content/en/docs/reference/controllers/daemonset.md

```diff
@@ -0,0 +1,102 @@
+---
+toc_hide: true
+title: DaemonSet controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The DaemonSet {{< glossary_tooltip term_id="controller" text="controller" >}} manages
+{{< glossary_tooltip term_id="pod" text="Pods">}} configured via a
+{{< glossary_tooltip term_id="daemonset" >}} object, to make sure that some
+(or all) {{< glossary_tooltip term_id="node" text="Nodes">}} run a copy of a
+Pod.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+The DaemonSet controller is built in to the {{< glossary_tooltip term_id="kube-controller-manager" >}}.
```

Comment:
```suggestion
The DaemonSet controller is built-in to the {{< glossary_tooltip term_id="kube-controller-manager" >}}.
```

Reply:
I'll see if I can reword to avoid having to pick a side (dash vs. no dash).

Reply:
How's this?